ui/raidboss/data/04-sb/raid/o6s.js

```diff
@@ -48,11 +48,12 @@
     },
     {
       id: 'O6S Storms Grip',
-      regex: / 03:\y{ObjectId}:Added new combatant The Storm's Grip/,
-      regexDe: / 03:\y{ObjectId}:Added new combatant Sturmgebiet/,
-      regexFr: / 03:\y{ObjectId}:Added new combatant Zone De Tempête/,
-      regexJa: / 03:\y{ObjectId}:Added new combatant 暴風域/,
-      regexKo: / 03:\y{ObjectId}:Added new combatant 폭풍 영역/,
+      regex: Regexes.addedCombatant({ name: 'The Storm\'S Grip', capture: false }),
```

Comment:
This is the tiniest of requests, but could you make this be "The Storm\'s Grip" without the capital S? (Same with other capital S below)

Reply:
done